src/source/feeds.rst

```diff
@@ -31,6 +82,701 @@ XML file.
+This field can only be set once during the lifecycle of an ad.
+
+An integer value from the category list. Must be an id of a category with a
+non-zero parent id.
+
+======= ====================================================
+Example	.. code-block:: html
+        
+         <ad:categoryId>945</ad:categoryId>
+======= ====================================================
+
+.. index:: status
+.. _feed_status:
+
+status
+""""""
+
+Ad status. One of the following:
+
+====== ====================================================
+Name   Description
+====== ====================================================
+ACTIVE The ad is active, it can be found on the website
+PAUSED The ad is paused, it cannot be found on the website
+====== ====================================================
+
+The provided (desired) status may differ from the resulting one, depending on the other conditions.
+For instance, budget may be depleted, or the seller may have too many active ads already in the category.
+
+======= =============================
+Example .. code-block:: html
+        
+         <ad:status>PAUSED</ad:status>
+======= =============================
+
+.. index:: url
+.. _feed_url:
+
+url
+"""
+
+An external URL that is shown on the ad page. This must be a valid http(s)
+url.
+
+======= =======================================
+Example .. code-block:: html
+        
+         <ad:url>https://www.bmw.de</ad:url>
+======= =======================================
+
+.. index:: vanityUrl
+.. _feed_vanityUrl:
+
+vanityUrl
+"""""""""
+
+The text/url that will be displayed instead of the url in :ref:`feed_url`.
+
+======= ==================================
+Example .. code-block:: html
+        
+         <ad:vanityUrl>BMW</ad:vanityUrl>
+======= ==================================
+
+.. index:: priceType
+.. _feed_priceType:
+
+priceType
+"""""""""
+
+Must be a valid price type identifier from the list of :ref:`price_types`.
+
+======= ============================================
+Example .. code-block:: html
+        
+         <ad:priceType>FIXED_PRICE</ad:priceType>
+======= ============================================
+
+.. index:: price
+.. _feed_price:
+
+price
+"""""
+
+The meaning of the value depends on the :ref:`feed_priceType`. 
+
+If it is `FIXED_PRICE` or `BIDDING_FROM` then **price** is mandatory and needs to be greater than 0.
+The maximum allowed **price** value is ``10000000000`` of the local market currency. (100.000.000,00 EUR / CAD / ... ).
+
+======= =============================
+Example .. code-block:: html
+        
+         <ad:price>1500</ad:price>
+======= =============================
+
+.. index:: originalPrice
+.. _feed_originalPrice:
+
+originalPrice
+""""""""""""""
+
+Product price before discount. Ignored if a seller does not have a discount option enabled. Must be greater than :ref:`feed_price`.
+The maximum allowed **originalPrice** value is ``10000000000`` of the local market currency. (100.000.000,00 EUR / CAD / ... ).
```

Comment:
wasn't it cents? or it differs from sellside API?

Reply:
Good question, afaik it was also about Euros. Considering we have only EU and Canada now, cents are fine, but perhaps it is more pragmatic to speak about "local market currency" without assuming there would be cents, not penny or anything else.